Box.V2/Managers/BoxFoldersManager.cs

```diff
@@ -173,11 +173,20 @@ public BoxFoldersManager(IBoxConfig config, IBoxService service, IBoxConverter c
         }
 
         /// <summary>
-        /// Used to create a shared link for this particular folder. Please see here for more information on the 
+        /// Used to create a shared link for this particular folder. Please see <see href="https://community.box.com/t5/Collaboration-and-Sharing/Shared-Links-Overview-And-FAQs/ta-p/142">here</see> for more information on the 
```

Comment:
should we use this link? https://docs.box.com/reference#create-a-shared-link-for-a-folder